zhmccli/_helper.py

```diff
@@ -1751,3 +1752,20 @@ def get_level_str(bundle_level, ftp_host):
     else:
         level_str = "all locally available firmware"
     return level_str
+
+
+def parse_yaml_flow_style(cmd_ctx, option_name, value):
```

Comment:
no mocked unit test for this?

Reply:
There is a unit test function `test_parse_yaml_flow_style()` for this function in tests/unit/test_helper.py.